### ISSUE TITLE
Info Outputs

### ISSUE THREAD

**Issue by swissglider** [clough42/node-red-contrib-thingspeak42#2]
Hi,
Is it possible that you change your many info outputs to debug.
Many thanks

**Comment by clough42** [comment]
Let me take a look.

**Comment by clough42** [comment]
Could you be a little more specific as to what you're asking?

Are you referring to the node.log() messages?  I'm aware of node.log(), node.warn() and node.error().  Or are you referring to something else?

**Comment by KillSwitchSIG** [comment]
Yes I agree with swissglider. There's too many node.log() in the code. This outputs too many messages to the NodeRed logs. If for example we are posting to thingspeak every 15secs this creates huge log files after a while.
